src/libs/ReportActionsUtils.js

```diff
@@ -325,6 +325,10 @@ function shouldReportActionBeVisible(reportAction, key) {
         return false;
     }
 
+    if (lodashGet(reportAction, 'message[0].moderationDecisions[0].decision') === CONST.MODERATION.MODERATOR_DECISION_PENDING_REMOVE) {
```

Comment:
Let's use moderationDecision here (we're migrating to that since we'll only return the last decision anyways)
https://github.com/Expensify/Auth/blob/87fb42c3df3638181e670b8b01d29922acc09d2e/auth/command/Get.cpp#L268-L268

Reply:
Hmm actually, I wonder if we'd have to update FlagComment then as well

Reply:
Actually yeah, I don't think this would work offline if we made this change - I added a clean up issue here https://github.com/Expensify/Expensify/issues/292452 but feel free to ignore the change here

Reply:
Ok cool thanks!